client/src/GlobalStyles.js

```diff
@@ -109,7 +109,20 @@ const GlobalStyles = () => {
           padding: 0 1em 0 1em;
         }
 
-        /* MAIN PAGE  */
+        /* MAP PAGE */
+
+        .main--map {
+          height: 60%;
+          flex: 1;
+          overflow: hidden;
+        }
+
+        .popup {
+          width: 50vw;
+          height: 20%;
+        }
```

Comment:
You define addition CSS classes.
Same issue like in the last pull request:
https://github.com/manuelsanchez2/doicheliving-app/pull/34#discussion_r488052384
You lose the advantages of styled components (see https://styled-components.com/docs/basics#motivation).

Are you sure, you need these styling anyway?

Reply:
You are right! Deleted! Thanks!